lib/smart_city/dataset.ex

```diff
@@ -45,7 +45,7 @@ defmodule SmartCity.Dataset do
       "sourceFormat": "",
       "sourceType": "",     // remote|stream|batch
       "cadence": "",
-      "sourceQueryParams": {
+      "sourceQueryParams": {  //queryParams is left in for backwards compatability, but will be removed in the future
```

Comment:
Should we do something like this?
```
source: %{
  queryParams: ...,
  headers: ...,
  url: ...
}
```

Reply:
Yeah, I do like that better/I think we should move in that direction.  I talked with Jarred though and we don't need the backwards compatibility at this point so I'm going to close this pull request.